core/src/main/resources/hudson/model/queue/_api.jelly

```diff
@@ -0,0 +1,17 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+
+  <h2>Canceling the build of an item in the Queue</h2>
+  <p>
+    To programmatically cancel the build of an item in the Queue, post to <a href="../cancelItem">this URL</a>.
+    The id of the item to cancel is expected.
```

Comment:
Expected where?

Reply:
Would rewording as:

"The id of the item to cancel is expected to be passed as a query parameter named id" answer your question?

Reply:
@PierreBtz That would work 👍